### DISCUSSION TITLE
Websocket Https Problemo

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#312] [Answers & Questions] [answered]
is there a way to circumvent this and be able to connect to an "insecure" websocket with ws://
cables is complaining...

![image](https://user-images.githubusercontent.com/1138673/184836093-ae624dd4-0d08-497c-8562-33ea2bf26669.png)

**Comment by wirmachenbunt** [comment]
btw. when i do so via the dev console of chrome with
`var webSocket = new WebSocket('ws://192.168.2.121:5555');
webSocket.onmessage = function(data) { console.log(data); }`
this works

**Reply by pandrr** [reply]
because console dev tools is not https.

**Comment by steam0r** [comment, marked as answer]
there is no way to circumvent that, you can disable the check in browsers for domains (if this is only for testing):

https://www.damirscorner.com/blog/posts/20210528-AllowingInsecureWebsocketConnections.html

**Comment by wirmachenbunt** [comment]
thanks guys, or i have to go through the pain of making the websocket secure on windows ...
doesn't matter too much, the ios electron cables app i'm building works with the insecure websocket.
cheers

**Reply by steam0r** [reply]
"localhost" is extempt from most of these security things also, if that helps...

**Reply by wirmachenbunt** [reply]
jo, thats what i use when developing locally. the workflow is ok, i switch the IP when i need to.
thanx

**Comment by pandrr** [comment]
you could open it in an electron app, like this:

https://github.com/pandrr/insecurebrowser

**Reply by wirmachenbunt** [reply]
ah nice, the famous insecure browser, thanks for sharing

**Reply by wirmachenbunt** [reply]
haha, standalone version of cables...there she is :)
